modules/las/src/lib/laslaz-decoder.ts

```diff
@@ -78,37 +96,66 @@ function parseLASHeader(arraybuffer) {
   o.maxs = [bounds[0], bounds[2], bounds[4]];
   o.mins = [bounds[1], bounds[3], bounds[5]];
 
-  return o;
+  return o as LASHeader;
 }
 
 // LAS Loader
 // Loads uncompressed files
 //
 class LASLoader {
-  constructor(arraybuffer) {
+  arraybuffer: ArrayBuffer;
+  readOffset: number;
+  header: LASHeader;
+
+  constructor(arraybuffer: ArrayBuffer) {
     this.arraybuffer = arraybuffer;
+    this.readOffset = 0;
+    this.header = {
+      pointsOffset: 0,
+      pointsFormatId: 0,
+      pointsStructSize: 0,
+      pointsCount: 0,
+      scale: [0, 0, 0],
+      offset: [0, 0, 0],
+      maxs: [0],
+      mins: [0],
+      totalToRead: 0,
+      totalRead: 0,
+      versionAsString: '',
+      isCompressed: true
+    };
   }
-
+  /**
+   * Nothing needs to be done to open this
```

Comment:
I think this comment should remain inside the function. This comment is not a description of the semantics of this function, but rather an implementation comment.